### PR TITLE
doc(AnnotatedCubeActor): add missing type annotations

### PR DESCRIPTION
**Pull request by yasushi-saito**
### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Context

`IO/Core/ImageStream/example` doesn't compile with typescript without this change.
This PR only changes the typescript definitions, not the actual js code.

### Changes

- [x] Documentation and TypeScript definitions were updated to match those changes

### Testing

- [ ] This change adds or fixes unit tests
- [ ] All tests complete without errors on the following environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**: 

_This contribution is funded by [Example](https://example.com)._

### CODE REVIEW

File: Sources/Rendering/Core/AnnotatedCubeActor/index.d.ts [hunk 1]
```diff
@@ -10,15 +10,16 @@ export interface IStyle {
 	fontSizeScale?: (res: number) => number;
 	edgeThickness?: number;
 	edgeColor?: string;
+	resolution: number;
```
Comment (daker) [inline Sources/Rendering/Core/AnnotatedCubeActor/index.d.ts:13]: can you make it optional with `?`

**Review by daker — CHANGES_REQUESTED** [review]

File: Sources/Rendering/Core/AnnotatedCubeActor/index.d.ts (hunk 1 above)
Comment (yasushi-saito) [reply Sources/Rendering/Core/AnnotatedCubeActor/index.d.ts]: done.

**Review by daker — APPROVED** [review]
LGTM

**Comment by finetjul** [comment]
Please make it a "doc" type of commit instead of "fix".

**Comment by yasushi-saito** [comment]
ok, done.

**Comment by finetjul** [comment]
Thanks for your contribution !

**Pull request merged**